docs/DeveloperGuide.md

```diff
@@ -273,13 +273,14 @@ interview availability dates.
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​  | I want to …​                          | So that I can…​                                                               |
-|----------|----------|---------------------------------------|-------------------------------------------------------------------------------|
-| `* * *`  | user     | add an applicant to the system        | keep track of their application details for consideration.                    |
-| `* * *`  | user     | delete an applicant from the system   | remove entries that I no longer need.                                         |
-| `* * *`  | user     | search for an applicant in the system | access details of the applicant without having to go through the entire list. |
-| `* * *`  | user     | list all applicants in the system     | monitor the application pool.                                                 |
-| `* * *`  | new user | view all available commands           | get familiarised with the system.                                             |
+| Priority | As a …​  | I want to …​                       | So that I can…​                                                               |
+|----------|----------|------------------------------------|-------------------------------------------------------------------------------|
+| `* * *`  | user     | add a candidate to the system      | keep track of their application details for consideration.                    |
+| `* * *`  | user     | delete a candidate from the system | remove entries that I no longer need.                                         |
+| `* * *`  | user     | find a candidate in the system     | access details of the candidate without having to go through the entire list. |
+| `* * *`  | user     | list all candidates in the system  | monitor the application pool.                                                 |
+| `*`      | user     | sort candidates                    | get familiarised with the system.                                             |
```

Comment:
Hi @charmainehly ! I was resolving my conflicts for #88 when I saw this. Do you want me to change `get familiarised with the system` for you? If so, what would you like me to change it to?